src/qibolab/qubits.py

```diff
@@ -41,11 +41,17 @@ class Qubit:
     readout_frequency: int = 0  # this is the dressed frequency
     drive_frequency: int = 0
     anharmonicity: int = 0
-    Ec: int = 0
-    Ej: int = 0
-    g: int = 0
+
+    sweetspot: float = 0.0
+    flux_to_bias: float = 0.0
+    asymmetry: float = 0.0
+    brf: float = 0.0
+    ssf_brf: float = 0.0
+    Ec: float = 0.0
+    Ej: float = 0.0
+    g: float = 0.0
```

Comment:
Could you please add some docstrings, below each definition, eg.:
```py
brf: float = 0.0
"""Explanation what brf is."""
```
that explain the new fields, particularly `brf` and `ssf_brf` as these may not be clear to everyone (at least they are not to me). Maybe `brf` and `ssf_brf` are also not the best names, as they are not very explanatory, but I understand we want to avoid very long names for usability.

Ideally we should add docstrings to all fields of the `Qubit` as this will improve the docs, but for the purpose of this PR we should at least add to the new fields.

Reply:
Could you also adopt naming conventions? `Ec` and `Ej` are capitalized as classes (and that's why GitHub is also highlighting them differently).

If you also had a more explicit name for `g` it would be nice.

Reply:
> Could you also adopt naming conventions? `Ec` and `Ej` are capitalized as classes (and that's why GitHub is also highlighting them differently).

`T1`, `T2` have the same issue but I guess we made an exception to follow the physics notation.

Reply:
@stavros11 thanks for your comments. I have added docstrings for the new attributes that are not clear. As you said, probably we should add docstrings for all the attributes, because there are some of them that are not clear what they are only with the attribute name.

Reply:
> > Could you also adopt naming conventions? `Ec` and `Ej` are capitalized as classes (and that's why GitHub is also highlighting them differently).
> 
> `T1`, `T2` have the same issue but I guess we made an exception to follow the physics notation.

I was thinking exactly the same. But I have already changed them to small letters. What do you want to do? To me the capital letters are more explanatory because we follow the physics notation... Also g, in superconducting physics is well known that is the readout coupling, so I have added the corresponding docstring but I think is better if we do not change the name

Reply:
> Special cases aren't special enough to break the rules.
Although practicality beats purity.
https://peps.python.org/pep-0020/

So, I know these exceptions are quite natural, but we do not have clear rules about when it is reasonable to make an exception and when it is not.
It is true that on the one side I am the one introducing the discussion, but my (limited) experience tells me that these issues come over and over again, so it is better just to establish a simple (as possible) rule and follow it.

So:
- if `g` is the readout coupling, I would rather call it `g_readout` or `readout_coupling`, because it might be unambiguous in a paper (or little ambiguous), but in a piece of code we have plenty of variables, so single letter ones are rather confusing
- for `Ec`, `T1` and friends I would use `ec`, `t1` and so on or more telling names as in the case of `g`

However, it is not a priority, the code will work anyhow. I just fear to be a bit more confusing in the long term, and then we won't like to change to avoid breaking compatibility.
Choose what you prefer :)

Reply:
If it is not clear, let's leave the Ec and Ej in capital letters, that is usual physics notation and to me is much more clear